### ISSUE TITLE
Importing SQL dump from app.monicahq.com has several issues

### ISSUE THREAD

**Issue by jonasvp** [monicahq/monica#1815]
**Describe the bug**

I tried importing the SQL dump from the hosted version on a docker image I'm running myself. The documentation (https://github.com/monicahq/monica/blob/master/docs/installation/update.md#importing-sql-from-the-exporter-feature) says:

> Then import monica.sql into your database. Tools like phpmyadmin or Sequel Pro might help you with that.

I finally managed to import the file but I had to:

1) Add `set foreign_key_checks=0;` at the top of the file
2) From the MySQL shell run `truncate changelogs, terms;` (those tables get exported as well, even though they're generic and get filled by Monica itself)
3) I had to replace `,key,` with `,'key',` - `key` is a reserved keyword and MySQL throws an error when it's not quoted.

Maybe this will help someone else - should I just add it to the documentation mentioned above?

**Comment by yeltsinxyz** [comment]
Same thing. But since I'm newbie on this, I will just create one by one again. 600 contacts one by one...

**Comment by chrisspiegl** [comment]
I also tried this and the main issue I noticed is that it does not seem to include the account details? I am not able to login on my local install.

**Comment by djaiss** [comment]
Alright let's see what's going on here.

@yeltsinxyz obviously, having to create a contact one by one sucks. We'll figure out what's the problem and solve it.

**Comment by chrisspiegl** [comment]
@djaiss I also was able to confirm the things that @jonasvp mentioned. The `,key,` problem especially as well as the not existing foreign keys. Combined with the problem that the account details do not seem to be included in the SQL file.

**Comment by Brice187** [comment]
The column `key` in TABLE `modules` has to be renamed, because in phpmyadmin you still get in trouble when trying to query the database with `'key'`instead of `key`.

My dirty trick: Rename column to something differently and change the occurences of `key` in .sql-file into this new string. After succesful completion you can rename the column back to `key`.

Next thing is to signup and if you do not see your contacts, you have to change your account_id in TABLE `users` to the correct one (**1** in most cases)

**Comment by IronTooch** [comment]
Also, minor thing (getting here from the Google), it seems like there was a Scheme change between version 2.9 and version 2.13 with respect to how addresses is handled. 

Exporting from 2.9 yields this statement: 

> INSERT INTO addresses (id,account_id,contact_id,name,street,city,province,postal_code,country,created_at,updated_at) VALUES (1,1,1,NULL,'123 Happy St','Philadelphia','PA',12345,'','2018-09-01 06:34:45','2018-09-01 06:34:45');

Which fails on import, because 2.13's schema for addresses does not have a street column. It appears to, instead, have a foreign key to the places table, where that sort of thing appears to live. This is in addition to the quoted key issue. 

I don't know how big a deal exporting is for users and moving several versions in a shot, but definitely a challenge.

**Comment by fmcpherson** [comment]
How do you get a mysql shell to do the import? So far I have only figured out how to connect mysql-workbench by exposing the port of mysql in docker-compose. A docker-compose run mysql doesn't seem to work or I am doing something wrong.

**Comment by fmcpherson** [comment]
I figured out how to install mysqlsh but not how to use it to import the data, so for now I am using the Import function of mysql workbench and I am getting the following error. The source and destination versions of monica are the same 2.13, using the dockerized version at both the source and destination. 

I edited the monica.sql created by the export function with the info jonasvp provided but I end up with the error below. I do not see a changelog table but I can truncate terms. I am following the instructions here: https://github.com/monicahq/monica/blob/master/docs/installation/update.md#importing-sql-from-the-exporter-feature including running php artisan migrate.

09:21:37 PM Restoring /home/frankmcpherson/monica/monica.sql
Running: mysql --defaults-file="/tmp/tmpBz1IBH/extraparams.cnf"  --protocol=tcp --host=100.115.92.202 --user=root --port=3306 --default-character-set=utf8 --comments --database=monica  < "/home/frankmcpherson/monica/monica.sql"
ERROR 1062 (23000) at line 236: Duplicate entry '1' for key 'PRIMARY'

Operation failed with exitcode 1
09:21:40 PM Import of /home/frankmcpherson/monica/monica.sql has finished with 1 errors
